spec/models/ingredients_alchemical_property_spec.rb

```diff
@@ -0,0 +1,323 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe IngredientsAlchemicalProperty, type: :model do
+  describe 'validations' do
+    let(:ingredient) { build(:ingredient) }
+
+    before do
+      create(:canonical_ingredient)
```

Comment:
This object isn't used in all specs. Can we move the `before` block into a context or two so this object isn't created every time?